src/sentry/integrations/jira/issue_hook.py

```diff
@@ -35,72 +34,77 @@ def get(self, request, issue_key, *args, **kwargs):
-
-        first_release = group.get_first_release()
-        if first_release is not None:
-            last_release = group.get_last_release()
-        else:
-            last_release = None
-
-        first_release_url = None
-        if first_release:
-            first_release_url = get_release_url(first_release)
-
-        last_release_url = None
-        if last_release:
-            last_release_url = get_release_url(last_release)
-
-        context = {
-            "type": result.get("metadata", {}).get("type", "Unknown Error"),
-            "title": group.title,
-            "title_url": get_group_url(group),
-            "first_seen": result["firstSeen"],
-            "last_seen": result["lastSeen"],
-            "first_release": first_release,
-            "first_release_url": first_release_url,
-            "last_release": last_release,
-            "last_release_url": last_release_url,
-            "stats_24hr": stats_24hr,
-            "stats_14d": stats_14d,
-        }
-
-        logger.info("issue_hook.response", extra=context)
-
-        return self.get_response(context)
+                stats = result["stats"][stats_period]
+                return result, reduce(accum, stats, 0)
+
+            def get_release_url(release):
+                project = group.project
+                return absolute_uri(
+                    "/organizations/{}/releases/{}/?project={}".format(
+                        project.organization.slug, quote(release), project.id
+                    )
+                )
+
+            def get_group_url(group):
+                return group.get_absolute_url(params={"referrer": "sentry-issues-glance"})
+
+            result, stats_24hr = get_serialized_and_stats("24h")
+            _, stats_14d = get_serialized_and_stats("14d")
+
+            first_release = group.get_first_release()
+            if first_release is not None:
+                last_release = group.get_last_release()
+            else:
+                last_release = None
+
+            first_release_url = None
+            if first_release:
+                first_release_url = get_release_url(first_release)
+
+            last_release_url = None
+            if last_release:
+                last_release_url = get_release_url(last_release)
+
+            context = {
+                "type": result.get("metadata", {}).get("type", "Unknown Error"),
+                "title": group.title,
+                "title_url": get_group_url(group),
+                "first_seen": result["firstSeen"],
+                "last_seen": result["lastSeen"],
+                "first_release": first_release,
+                "first_release_url": first_release_url,
+                "last_release": last_release,
+                "last_release_url": last_release_url,
+                "stats_24hr": stats_24hr,
+                "stats_14d": stats_14d,
+            }
+
+            logger.info("issue_hook.response", extra=context)
```

Comment:
You won't want all this data going to logs in production.

Reply:
oh good catch, I didn't add this myself